### PR TITLE
Modified beatles automl

### PR DESCRIPTION
**Pull request by stevenpais1**
Hi Richard - Please review

Approached used

awesome data science beatles-automl  had several missing pieces. non working links, missing files etc...Leveraged Brian's github (https://github.com/brianray/beatles_machine_learning_data). Verified. It looked complete from all angles - used it as a single source of truth.

Kept create_training_data.ipynb which was missing in brian's github but present in awesome data science beatles-automl answers folder. This script creates file_out_2495_tags.csv

links should be working correctly. All missing files should now be present

### CODE REVIEW

File: beatles-automl/README.md [hunk 1]
```diff
@@ -1,27 +1,22 @@
...
 
-### quickstart
-
-
 Data for use on Machine Learning Model to predict those who will listen The Beatles based on other artists. There are two data sets of importance:
 
-* [file_out_2495.csv](./answers/file_out_2495.csv) a list of users who listened to at least 1 of the most 300 played artists. The columns are the play counts for each artists mentioned. The target is "Likes the Beatles"
-* [file_out_2495_tags.csv](./answers/file_out_2495_tags.csv) Same a above but with also a count of the genre distribution. 
-
+* [file_out_2495.csv](./file_out_2495.csv) a list of users who listened to at least 1 of the most 300 played artists. The columns are the play counts for each artists mentioned. The target is "Likes the Beatles"
```
Comment (truongc2) [inline beatles-automl/README.md:5]: Add the "answers" directory in the hyper link

File: beatles-automl/README.md [hunk 2]
```diff
@@ -1,27 +1,22 @@
...
+* [file_out_2495_tags.csv](./file_out_2495_tags.csv) Same a above but with also a count of the genre distribution.
```
Comment (truongc2) [inline beatles-automl/README.md:6]: Add the "answers" directory in the hyper link

**Comment by truongc2** [comment]
Added some comments for the hyperlinks. Also, can you remove the License as we have one for the main project.

File: beatles-automl/README.md (hunk 1 above)
Comment (stevenpais1) [reply beatles-automl/README.md]: Added. Please check.

File: beatles-automl/README.md (hunk 2 above)
Comment (stevenpais1) [reply beatles-automl/README.md]: Added. Please check.

**Comment by stevenpais1** [comment]
> Added some comments for the hyperlinks. Also, can you remove the License as we have one for the main project.

yes done. please check.

**Comment by stevenpais1** [comment]
Hi Richard - Please note.

* [file_out_2495_tags.csv](./answers/file_out_2495_tags.csv) link will not work in current state as this file was missing. Once you approve the merge pull request. File should be placed. So link will then work.

**Review by truongc2 — APPROVED** [review]
LGTM

**Pull request merged**